src/components/timeline/ActiveTabGlobalTrack.js

```diff
@@ -152,7 +150,11 @@ class ActiveTabGlobalTrackComponent extends PureComponent<Props> {
 // Provide an empty list, so that strict equality checks work for component updates.
 const EMPTY_RESOURCE_TRACKS = [];
 
-export default explicitConnect<OwnProps, StateProps, DispatchProps>({
+export const ActiveTabGlobalTrackComponent = explicitConnect<
```

Comment:
Can you please remove the word `Component` from all the exported names? No need to look at the internal names, I'm concerned only about the exported ones.